docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -63,6 +68,28 @@ status: {}
 More information about the [scale subresource can be found in the Kubernetes
 documentation][scale].
 
+#### optional `spec` fields
+
+* `machineTemplate` - is a struct containing details of the control plane 
+  machine template.
+
+* `machineTemplate.metadata` - is a struct containing info about metadata for control plane
+  machines.
+
+* `machineTemplate.metadata.labels` - is a map of string keys and values that can be used 
+  to organize and categorize control plane machines.
+
+* `machineTemplate.metadata.annotations` - is a map of string keys and values containing
+  arbitrary metadata to be applied to control plane machines.
+
+* `machineTemplate.infrastructureRef` - is a corev1.ObjectReference to a custom resource
+  offered by an infrastructure provider. The namespace in the ObjectReference should
```

Comment:
Should we maybe use `must` instead of `should` here?

Reply:
done!